packages/salesforcedx-vscode-core/package.json

```diff
@@ -443,18 +470,34 @@
         "command": "sfdx.force.lightning.app.create",
         "title": "%force_lightning_app_create_text%"
       },
+      {
+        "command": "sfdx.internal.lightning.app.create",
+        "title": "%force_lightning_app_create_text%"
```

Comment:
its unclear to me why these have to be duplicated w/ a separate command id? I thought they could be the same command and internally we just check a setting as to what parameters to pass...

Reply:
I want to have a clear separation between how we run a internal command vs a normal one. That meant creating an additional entry point where I can define preconditions, param gatherers and postcondition methods that are exclusive to the internal mode e.g. `forceInternalLightningInterfaceCreate` in `forceLightningInterfaceCreate.ts`.